### PR TITLE
Squad Update

### PR DESCRIPTION
**Pull request by southnode**
Update to add fixedmaxtickrate to start arguments, remove haswritableconsole (Squad does not have a writable console), add SteamWorkshopDownloadLocation to applicable file area, set minampversion, remove RCON from serverconfig area and replace with info tile, clear up verbiage for descriptions of settings

### CODE REVIEW

File: squad-dedicated-server.kvp [hunk 1]
```diff
@@ -4,7 +4,7 @@ App.ApplicationPort2=27165
 App.ApplicationPort3=0
 App.ApplicationReadyMode=RegexMatch
 App.BaseDirectory=./squad-dedicated-server/403240/
-App.CommandLineArgs=MULTIHOME={{$ApplicationIPBinding}} Port={{$ApplicationPort1}} QueryPort={{$ApplicationPort2}} RCONPORT={{$RemoteAdminPort}} RCONPASSWORD={{$RemoteAdminPassword}} -log -fullcrashdump
+App.CommandLineArgs=MULTIHOME={{$ApplicationIPBinding}} Port={{$ApplicationPort1}} QueryPort={{$ApplicationPort2}} RCONPORT={{$RemoteAdminPort}} RCONPASSWORD={{$RemoteAdminPassword}} FIXEDMAXTICKRATE={{$MaxTPS}} -log -fullcrashdump
```
Comment (IceOfWraith) [inline squad-dedicated-server.kvp:7]: Take off the $ on MaxTPS here and in the setting. That's reserved for AMP variables.

File: squad-dedicated-serverconfig.json [hunk 2]
```diff
@@ -37,10 +37,24 @@
         "Suffix":"players",
         "MinValue":"1"
     },
+    {
+        "DisplayName": "Max TPS",
+        "Category": "Server Config",
+        "Description": "Set the maximum tickrate for the server (Set below 45 for best stability)",
+        "Keywords": "tps, tickrate",
+        "FieldName": "$MaxTPS",
```
Comment (IceOfWraith) [inline squad-dedicated-serverconfig.json:45]: $

File: squad-dedicated-serverconfig.json [hunk 3]
```diff
@@ -228,7 +242,7 @@
     {
         "DisplayName": "Record Demos",
```
Comment (IceOfWraith) [inline squad-dedicated-serverconfig.json:243]: Just a recommendation, should this be renamed to reflect server side like the description?

**Review by IceOfWraith — CHANGES_REQUESTED** [review]
Thank you for this as well! I've provided some feedback. You clearly know how the game works, so I won't second guess any of the changes. Lol

File: squad-dedicated-server.kvp (hunk 1 above)
Comment (southnode) [reply squad-dedicated-server.kvp]: Fixed

File: squad-dedicated-serverconfig.json (hunk 2 above)
Comment (southnode) [reply squad-dedicated-serverconfig.json]: Removed

File: squad-dedicated-serverconfig.json (hunk 3 above)
Comment (southnode) [reply squad-dedicated-serverconfig.json]: Have reworded both demo settings for better readability and descriptiveness

**Review by IceOfWraith — APPROVED** [review]
Nice job!

**Pull request merged**